src/ui/public/vis_maps/kibana_map.js

```diff
@@ -0,0 +1,507 @@
+import { EventEmitter } from 'events';
```

Comment:
The same zoom issue on save is still there as well. If you zoom in a bunch, and then hit save, it zooms you out all the way.

![tilemap-zoom-on-save](https://cloud.githubusercontent.com/assets/627123/24294624/b83750d6-106d-11e7-81ef-4bfaba0e3823.gif)